packages/components/src/button/index.stories.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import Button from '.';
+
+export default { title: 'Button' };
+
+const helloWorld = `Hello World!`;
+const handleClick = action( 'click' );
+
+export const Default = () => <Button onClick={ handleClick }>{ helloWorld }</Button>;
+export const IsPrimary = () => (
+	<Button primary onClick={ handleClick }>
+		{ helloWorld }
+	</Button>
+);
```

Comment:
When clicking on the "Is Primary" option in the storybook, the expected styles are not applied. Can Storybook deal with styles imported as `import './style.scss'`? Does the webpack config that it uses support the `scss-loader` and `style-loader`?

Reply:
I've already added [support for SCSS](https://github.com/Automattic/wp-calypso/pull/38496/files#diff-2d3e9f995aea7d9f08deedc8d0b1087bR1) and you can see the black outline styles being applied in the screenshot above.

I (purposely) haven't imported the Calypso base styles where the CSS variables and other styles are being applied - the reason being that I want to develop and test these components as other A12s would find them without pulling in the Calypso base styles.

Reply:
I see, the button styles are present, but the theme CSS that turns CSS variables into actual colors is not.

Reply:
Yep, that's intentional in order to demonstrate the isolated use of `@automattic/components`.